model/project_parser.go

```diff
@@ -589,7 +588,28 @@ func LoadProjectInto(ctx context.Context, data []byte, opts *GetProjectOpts, ide
 				opts.ReadFileFrom = ReadFromPatch
 			}
 		}
-		yaml, err := retrieveFile(ctx, *opts)
+		var yaml []byte
+		if path.Module != "" {
+			module, err := GetModuleByName(intermediateProject.Modules, path.Module)
+			if err != nil {
+				return nil, errors.Wrapf(err, "can't get module for module name '%s'", path.Module)
+			}
+			repoOwner, repoName := module.GetRepoOwnerAndName()
+			moduleOpts := GetProjectOpts{
+				Ref: &ProjectRef{
+					Id:    module.Name,
+					Owner: repoOwner,
+					Repo:  repoName,
+				},
+				RemotePath:   path.FileName,
+				Revision:     module.Branch,
+				Token:        opts.Token,
+				ReadFileFrom: ReadfromGithub,
```

Comment:
We will eventually need to figure out how to handle the other cases but for now we can do just this one.

Reply:
checks for local modules first